### DISCUSSION TITLE
Help with Frigate/Double Take Automation

### DISCUSSION THREAD

**Discussion by chench00** [blakeblackshear/frigate#5245] [Ask A Question] [answered]
Since Frigate and Doubletake works together, I hope it's okay to ask for help here. 

I am trying to create a notification to unlock my side door based on the detection of my wife to or I arriving and our phones connecting to Wifi (as a security measure) on the condition that the door is already locked and doubletake detects one of us but I feel like I am missing something on the condition.

Here's the automation:

    - alias: Double Take - Door Unlock
      trigger:
        - platform: state
          entity_id: device_tracker.ME
          to: 'home'
        - platform: state
          entity_id: device_tracker.WIFE
          to: 'home'

      condition:
        condition: and
        conditions:
          - condition: state
            entity_id: lock.kitchen_door
            state: locked
          - condition: or
            conditions:
              - condition: state
                entity_id: sensor.double_take_ME
              - MISSING SOMETHING HERE? <------------------
              - condition: state
                entity_id: sensor.double_take_WIFE
              - MISSING SOMETHING HERE? <------------------

      action:
        - service: lock.unlock
          entity_id: lock.kitchen_door

        - service: notify.ios
          data:
            message: |-
              {% if trigger.to_state.attributes.match is defined %}
                {{trigger.to_state.attributes.friendly_name}} is near the {{trigger.to_state.state}} @ {{trigger.to_state.attributes.match.confidence}}%. Door has been unlocked.
              {% endif %}
            data:
              attachment:
                url: |-
                  {% if trigger.to_state.attributes.match is defined %}
                    http://192.168.143.11:3000/api/storage/matches/{{trigger.to_state.attributes.match.filename}}?box=true
                  {% endif %}
              push:
                sound: "3rd_party_critical.caf"

              actions:
                - action: URI
                  title: View Image
                  uri: |-
                    {% if trigger.to_state.attributes.match is defined %}
                      http://192.168.143.11:3000/api/storage/matches/{{trigger.to_state.attributes.match.filename}}?box=true
                    {% endif %}
      mode: parallel
      max: 10
      max_exceeded: silent

I believe I am missing something near the arrows to trigger the state?

**Comment by NickM-27** [comment, marked as answer]
are you expecting that the location will always update AFTER doubletake has detected your face? To me it seems like your location would probably update before that happened.

**Comment by chench00** [comment]
Actually, you are right. When the trigger fires most likely I would still be in my car so doubletake wouldn't have detected my face yet. I think I am better off using a condition.

Thanks for pointing that out.

**Comment by chench00** [comment]
While this thread is open, @NickM-27 mind if I ask if you have any idea why a doubletake automation would fire two notifications? I am using something similar to what you posted on another thread here. I can paste it here if needed.

**Reply by NickM-27** [reply]
it depends what you're using as the doubletake trigger, but it will update for every detection including the detection where it found a match

**Reply by chench00** [reply]
I am simply using:

        - alias: Double Take
          trigger:
            - platform: state
              entity_id: sensor.double_take_ME
